docs/VersionMigration.md

```diff
@@ -9,3 +9,34 @@ Updating to this release will not require any code changes for most SDK installa
 ### Removed
 
 - To simplify the public API, `ExperienceAction` and `ExperienceTrait` related classes and functions no longer have public visibility. There are no current use cases for extensibility that require these to be public. This change impacts items in the `com.appcues.action` and `com.appcues.trait` packages.
+
+
+# 1.x to 2.0 Migration Guide
```

Comment:
fwiw, on the iOS side we had a single `#` header and then each version within was a `##` sub-header, kind of liked that.

https://github.com/appcues/appcues-ios-sdk/pull/442/files

Reply:
Only did that because the DocC compiler complained, but I agree that it's usually best practice to have a single <h1> in a doc.